UCSLibraries/src/main/java/it/cnr/iit/xacml/wrappers/PolicyWrapper.java

```diff
@@ -253,12 +272,12 @@ private RuleType copyRuleType( RuleType ruleType,
     }
 
     /**
-     * Performs a copy of the policyType object.
+     * Performs a partial copy of the policyType object.
      *
      * @return the PolicyType object that is the copy of the one stored in this
      *         object
      */
-    private PolicyType clonePolicyType() {
+    private PolicyType almostClonePolicyType() {
```

Comment:
--> partiallyClonePolicyType() ?
Also why almost/partially? Just so that I can understand it

Reply:
yes, it's the way it is.
In practice the getPolicyForCondition() creates another policy from UXACML to XACML.
this function copies everything but the rule part that is inserted afterwards depending on the status
try/start/end access

Reply:
OK in that case, how about clonePolicyTypeWithoutRuleOrCondition() ???

Reply:
yes! 
ok done!